cogs/birthday.py

```diff
@@ -49,7 +52,17 @@ async def list(self, ctx, query=""):
 
             if user is not None:
                 text += "User: {}\t->\t{}.{}.\n".format(user.display_name, result[1], result[2])
```

Comment:
Just calculate in here instead of resplitting the lines later. A early return in case `user` is None should help with readability.

Reply:
👍 great idea.. did not think about that

As I understand the doc, if a user would leave a server, he would not be found.. thus nonvisible breaking this sysem, if using a simple return

I added a if-else system that does visibaly break and show on what the fault happend 
+it still continues to work

correct me if I'm wrong on this